pkg/riff/commands/application.go

```diff
@@ -24,7 +24,7 @@ import (
 func NewApplicationCommand(c *cli.Config) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "application",
-		Short:   "<todo>",
+		Short:   "build applications from source using Cloud Foundry buildpacks",
```

Comment:
in this case, Cloud Foundry is the providing the buildpacks used. Cloud Native Buildpacks describes the contract those buildpacks are implemented with. The builder image is `cloud foundry/cnb:bionic`.

You're right that this may not always be true if a custom builder is installed.